website/templates/project/project_base.mako

```diff
@@ -14,6 +14,11 @@
 
 <%include file="project_header.mako"/>
 
+% if status and node:
+    ## WARNING: Need new heading alert messages? They must be added at both here and base.mako
+    <%include file="../alert.mako"/>
```

Comment:
@sloria Yes, in `project_base.mako`, it is guaranteed to be defined -- See the code link here -- https://github.com/haoyuchen1992/osf.io/blob/jira/fix-project-menu-mobile/website/templates/project/project_base.mako#L76